src/datastore/postgres-store.ts

```diff
@@ -2665,6 +2711,176 @@ export class PgDataStore extends (EventEmitter as { new (): DataStoreEventEmitte
     return { results };
   }
 
+  async updateNames(bnsName: DbBNSName) {
```

Comment:
This function should accept a `client` as the first argument, i.e.
```ts
async updateName(client: ClientBase, bnsName: DbBNSName) { ... }
```
Then be called from the `async update(data: DataStoreUpdateData) { ... }` where it is already in a sql transaction.

Reply:
Added client as the first argument.